fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneRecordContextProperties.java

```diff
@@ -112,4 +112,8 @@ public final class LuceneRecordContextProperties {
      * assume that merge will not be performed during IO but in a background process.
      */
     public static final RecordLayerPropertyKey<Boolean> LUCENE_USE_CONCURRENT_MERGE_SCHEDULER = RecordLayerPropertyKey.booleanPropertyKey("com.apple.foundationdb.record.lucene.useConcurrentMergeScheduler", false);
+    /**
+     * Use "default transaction priority" during merge. The default of this prop is "true" because merge over multiple transactions may be holding back user operations.
```

Comment:
```suggestion
     * Use "default transaction priority" during merge. The default of this prop is {@code true} because merge over multiple transactions may be preventing user operations.
```